### ISSUE TITLE
Regarding the criterion used for the selection of best epoch

### ISSUE THREAD

**Issue by Rohan-IITG** [adbailey1/DepAudioNet_reproduction#9]
Hi,

Thank you for sharing the code. It helped in my research a lot.

Please share the reference that you refereed for selecting the criterion (train_f/4 - train_loss/10 + dev_f - dev_loss/10).
In case you selected the criterion on your own, can you please explain why you chose it instead of choosing the minimum val loss or maximum f-score criterion?

with regrads,
Rohan

**Comment by adbailey1** [comment]
Hi Rohan, 

My pleasure, I am glad it's helped!

I created that equation myself to make sure that the "best" validation epoch was always chosen with respect to the training score. Due to the volatile nature of the data/model, sometimes the validation score was extremely large at the start of training despite the training score being very poor. This equation was created to fix that. 

However, this was an early bit of code that I wrote and it worked for my usage but you are right, you could explore simply evaluating at the lowest validation loss. Again, due to volatility, you may want to search around the lowest validation loss space to find the best epoch.

Please feel free to change the code as you see fit. 

Kind regards, 

Andrew

**Comment by Rohan-IITG** [comment]
Thank you for your quick response. It makes sense to me.
